aliases/available/kubectl.aliases.bash

```diff
@@ -18,7 +18,7 @@ function _set_pkg_aliases()
     alias kcdn='kubectl describe node'
     alias kcgpan='kubectl get pods --all-namespaces'
     alias kcgdan='kubectl get deployments --all-namespaces'
-    alias kcnetshoot='kubectl run --generator=run-pod/v1 netshoot-$(uuidgen | tr A-Z a-z | sed 's/-//g') --rm -i --tty --image nicolaka/netshoot -- /bin/bash'
+    alias kcnetshoot='kubectl run --generator=run-pod/v1 netshoot-$(date +%s) --rm -i --tty --image nicolaka/netshoot -- /bin/bash'
```

Comment:
Since this is a fairly complex command, can you please add a comment above the alias describing what this does?

This will make it easier for other people to understand what this is doing...

Reply:
✅